tests/sentry/search/events/builder/test_metrics.py

```diff
@@ -2055,54 +2094,89 @@ def test_query_normal_distribution(self):
 
 
 class AlertMetricsQueryBuilderTest(MetricBuilderBaseTest):
+    def test_standard_metrics(self):
+        query = AlertMetricsQueryBuilder(
+            self.params,
+            use_metrics_layer=False,
+            granularity=3600,
+            query="",
+            dataset=Dataset.PerformanceMetrics,
+            selected_columns=["p75(measurements.fp)"],
+            on_demand_metrics_enabled=True,
+        )
+
+        assert query.on_demand_metrics_enabled
+        assert not query._on_demand_spec
```

Comment:
This tests class internals, is there some public effect we could observe instead? Such as a call to a (mocked) utility or a flag returned?